Common/DTO/CreateBrokerData.cs

```diff
@@ -8,11 +8,11 @@ public class CreateBrokerData
         /// <summary>
```

Comment:
Can we please rename this class? CreateBrokerData is action . And DTO it is some object. I think it is misleading

Reply:
> Can we please rename this class? CreateBrokerData is action . And DTO it is some object. I think it is misleading

Corrected.